src/App.jsx

```diff
@@ -1,104 +1,26 @@
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
+
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+const posts = postsFromServer.map(post => {
```

Comment:
It's a nice idea to prepare data before passing it into components

Reply:
Thank you!)